plugins/modules/aci_l3out.py

```diff
@@ -307,26 +386,84 @@ def main():
     enforceRtctrl = module.params.get("route_control")
     vrf = module.params.get("vrf")
     l3protocol = module.params.get("l3protocol")
+    ospf = module.params.get("ospf")
     asn = module.params.get("asn")
     state = module.params.get("state")
     tenant = module.params.get("tenant")
     name_alias = module.params.get("name_alias")
 
     if l3protocol:
         if "eigrp" in l3protocol and asn is None:
-            module.fail_json(msg="Parameter 'asn' is required when l3protocol is 'eigrp'")
+            module.fail_json(
+                msg="Parameter 'asn' is required when l3protocol is 'eigrp'"
+            )
         if "eigrp" not in l3protocol and asn is not None:
```

Comment:
Do we need similar warning for the ospf dict when not provided as protocol?

Reply:
I think for the fail message we do. But I am not sure about the warning message